<robot name="ur10">
  <gripper name="gripper" clearance="0.08">
    <position xyz="0.25 0 0" xyzw="0 0 0 1"/>
    <link name="ee_link"/>
  </gripper>

  <disable_collisions link1="shoulder_link"
		      link2="upper_arm_link" reason=""/>
  <disable_collisions link1="upper_arm_link"
		      link2="forearm_link" reason=""/>
  <disable_collisions link1="forearm_link"
		      link2="wrist_1_link" reason=""/>
  <disable_collisions link1="wrist_1_link"
		      link2="wrist_2_link" reason=""/>
  <disable_collisions link1="wrist_2_link"
		      link2="wrist_3_link" reason=""/>
  <disable_collisions link1="shoulder_link"
		      link2="forearm_link" reason=""/>
  <disable_collisions link1="wrist_1_link"
		      link2="wrist_3_link" reason=""/>
</robot>
